news/provision-newsfeed.sh

```
@@ -0,0 +1,23 @@
+  echo "Must specify docker image as first argument"
+fi
+
+DOCKER_IMAGE=$1
+
+echo "Provisioning docker image $DOCKER_IMAGE"
+
+# cleanup previous deployment
+docker stop newsfeed || true
+docker rm newsfeed || true
+
+eval $(aws ecr get-login --region eu-west-1 --no-include-email)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)